File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/09_compito_java_20160615/Pallavolo.py
from Squadra import Squadra


class Pallavolo(Squadra):
    def __init__(self, nome, codice, sport, vinte, perse, numero_medio_set_vinti):
        super().__init__(nome, codice, sport, vinte, perse)
        self._numero_medio_set_vinti = numero_medio_set_vinti

    def __str__(self):
        return super().__str__() + "\t-\t" + str(self._numero_medio_set_vinti) + "\t" + self._sport

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/09_compito_java_2016_06_15/Pallavolo.java
public class Pallavolo extends Squadra {
    private float setVintiMedia;

    public Pallavolo(String nome, int codice, String sport, int vinte, int perse, float setVintiMedia) {
        super(nome, codice, sport, vinte, perse);
        this.setVintiMedia = setVintiMedia;
    }

    public String toString() {
        return super.toString() + "\t-\t" + setVintiMedia + "\t" + "pallavolo";
    }
}